test/propagated_params_test.go

```diff
@@ -0,0 +1,161 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2019 The Tekton Authors
```

Comment:
nit: year to be updated

Reply:
😁 Thats what happens when you copy a file and start replacing its contents. Will fix

Reply:
done